content/2017.03.building-projects-as-a-distributed-team.rst

```diff
@@ -0,0 +1,251 @@
+Building software as a distributed team
+#######################################
+
+:slug: building-software-as-a-distributed-team
+:date: 2017-03-15
+:lang: en
+:url: building-software-as-a-distributed-team
+:summary:
+    Having multiple people working on a project is not as easy as it seems.
+	Here are some elements to move projects forward as a team.
+
+
+Today, I realized that I've been working for Mozilla for three years
+already.
+
+I've learned a lot, and with the storage team we have been part of a
+number of different projects:
+
+- Firefox Hello: Building a WebRTC signaling service.
+- MSISDN auth: Building a password less BrowserID authentication service using a phone number.
+- Syncto: Browsing Firefox Sync using the Kinto protocol.
+- Kinto: Building synchronization services with backup and sharing capability in mind.
+- RemoteServiceWorker: Building a Firefox Service Worker in the cloud
+- WebPush channels: Building a PubSub on top of WebPush
+
+While most of these projects have been built, deployed in production
+and eventually decommissined, some of them are still actively use by
+Firefox users everyday.
+
+
+How do you keep motivated building things that are thrown away?
+===============================================================
+
+Let's be honest, everything we build: cars, houses, and even softwares
```

Comment:
I'd move the colon earlier in the sentence: *Let's be honest: everything we build, cars, ...*